predicators/main.py

```diff
@@ -228,11 +229,15 @@ def _generate_interaction_results(
             request.train_task_idx,
             request.termination_function,
             max_num_steps=CFG.max_num_steps_interaction_request,
+            init_state=last_state
```

Comment:
can we move this out for better readability? so `init_state = CFG.online_learning_use_last_state else None` above `utils.run_policy` and then just `init_state=init_state` here